src/player.js

```diff
@@ -245,6 +245,12 @@ export default class Player extends FakeEventTarget {
    * @private
    */
   _playbackStart: boolean;
+  /**
+   * The available playback rates for the player.
+   * @type {Array<numbers>}
+   * @private
+   */
+  _playbackRates: Array<numbers>;
```

Comment:
no such thing `numbers`, `Array<number>`, also in JSDoc
Also in the `.md` file

Reply:
@OrenMe 
Fixed and checked with yarn flow as well.